CodeReview/Amie_code.py

```diff
@@ -21,14 +24,22 @@ def f(data):
     data['Minute'] = pd.DatetimeIndex(data['Date Time']).minute
     data['Time'] =  data['Hour'] + data['Minute']/60
     return data
+
 data1 = f(data)
+#Only grab 2014 data
```

Comment:
There is definitely an easier way to do this. You can set the index to data["Date Time"] and then use data.loc["startdate":"enddate"] where "startdate" and "enddate" follow the format of the index.  e.g. ["2014-01-01 9:12:00":"2014-01-01 9:36:00"]